### PR TITLE
Remove external kubernetes proxy mode

### PR DESCRIPTION
**Pull request by sttts**
Proxy mode was deprecated in 3.5 and will be removed for 3.6.

### CODE REVIEW

**Comment by deads2k** [comment]
lgtm [merge]

File: test/integration/service_account_test.go [hunk 1]
```diff
@@ -107,6 +107,10 @@ func TestServiceAccountAuthorization(t *testing.T) {
 		t.Fatalf("error creating kubeconfig: %v", err)
 	}
 
+	// TODO(rebase): clarify whether we need anything of the following anymore if we drop proxy-only mode @liggitt
+	t.Logf("Skipping remaining external-kube based test")
```
Comment (sttts) [inline test/integration/service_account_test.go]: @liggitt can you comment?

**Comment by deads2k** [comment]
@sttts some sort of verify error.

**Comment by sttts** [comment]
[test]

**Comment by deads2k** [comment]
re[test]

**Comment by sttts** [comment]
re[test]

**Comment by deads2k** [comment]
@sttts looks like a real failure.  At least one test timeout which is unusual in origin.  I'm trying to figure out where we can write up bugs against our CI.

**Comment by sttts** [comment]
This was part of the apigroups PR. Closing.

**Pull request closed without merging**